lib/plugins/install/install.js

```diff
@@ -67,23 +67,40 @@ class Install {
     ].join('');
 
     const servicePath = path.join(process.cwd(), parsedGitHubUrl.repo);
-
-    // throw an error if service path already exists
-    if (this.serverless.utils.dirExistsSync(servicePath)) {
-      const errorMessage = `A folder named "${parsedGitHubUrl.repo}" already exists.`;
-      throw new this.serverless.classes.Error(errorMessage);
+    const endIndex = parts.length - 1;
+    let repoPath;
+    if (parts.length > 4) {
+      repoPath = path.join(process.cwd(), parts[endIndex]);
+      if (this.serverless.utils.dirExistsSync(parts[endIndex])) {
+        const errorMessage = `A folder named "${parts[endIndex]}" already exists.`;
+        throw new this.serverless.classes.Error(errorMessage);
+      }
+      this.serverless.cli.log(`Downloading and installing "${parts[endIndex]}"...`);
+    } else {
+      if (this.serverless.utils.dirExistsSync(servicePath)) {
+        const errorMessage = `A folder named "${parsedGitHubUrl.repo}" already exists.`;
+        throw new this.serverless.classes.Error(errorMessage);
+      }
+      this.serverless.cli.log(`Downloading and installing "${parsedGitHubUrl.repo}"...`);
     }
-
-    this.serverless.cli.log(`Downloading and installing "${parsedGitHubUrl.repo}"...`);
     const that = this;
-
     // download service
     return download(
       downloadUrl,
       servicePath,
       { timeout: 30000, extract: true, strip: 1, mode: '755' }
     ).then(() => {
-      that.serverless.cli.log(`Successfully installed "${parsedGitHubUrl.repo}".`);
+      if (parts.length > 4) {
+        let dirctory = servicePath;
+        for (let i = 5; i <= endIndex; i++) {
+          dirctory = path.join(dirctory, parts[i]);
+        }
+        that.serverless.utils.copyDirContentsSync(dirctory, repoPath);
```

Comment:
`that.serverless.utils.copyDirContentsSync(dirctory, path.join(process.cwd(), parts[endIndex]));`